src/components/GenerateCVButton/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { PropTypes as PT } from 'prop-types';
+import buttonTypes from 'Constants';
+import { Button } from 'Common';
+import container from './container';
+
+const GenerateButton = ({ disabled, handleClick }) => {
+  return (
+    <Button
+      value="Generate CV"
+      disabled={disabled}
+      type={buttonTypes.submit}
+      handleClick={handleClick}
+    />
+  );
+};
+
+export default container(GenerateButton);
+
+GenerateButton.propTypes = {
```

Comment:
Just one of those personal things I like to have the export at the bottom. Basically everything in the component is above it before saying "I'm ready to be exported"